### PR TITLE
Fixed gazebo simulation and added depth camera

### PR DESCRIPTION
**Pull request by JosephSaliba01**
Maybe only review changes done to `drive_control/test/*`, other than that all should be good.

### CODE REVIEW

**Review by acoulombe — APPROVED** [review]
Aside from the odd/somewhat descriptive dae file names, everything looks good

File: drive_control/src/drive_control_node.py [hunk 1]
```diff
@@ -3,37 +3,64 @@
...
+        """
+        Hard-coded the simulation rover measurement, will need to adapt it towards the real model.
+        Drive control takes care interpolating velocities and publishing them.
+        """
+        self.wheel_radius = 0.04688
+        self.wheel_base_length = 0.28
         self.wheel_speed = None
         self.correction_wheel_speed = None
         self.steering = Steering(self.wheel_radius, self.wheel_base_length)
         self.right_front_pid_controller = PID(1, 0.1, 0.05, 0)
```
Comment (acoulombe) [inline drive_control/src/drive_control_node.py:27]: Remove these. Deprecated

File: drive_control/src/drive_control_node.py [hunk 2]
```diff
@@ -3,37 +3,64 @@
 currentdir = os.path.dirname(os.path.realpath(__file__))
 sys.path.append(currentdir)
 import rospy
+import numpy as np
 from steering import Steering
 from simple_pid import PID
```
Comment (acoulombe) [inline drive_control/src/drive_control_node.py:8]: Remove Line Deprecated

File: drive_control/src/drive_control_node.py [hunk 3]
```diff
@@ -3,37 +3,64 @@
...
         self.wheel_speed = None
         self.correction_wheel_speed = None
         self.steering = Steering(self.wheel_radius, self.wheel_base_length)
         self.right_front_pid_controller = PID(1, 0.1, 0.05, 0)
         self.left_front_pid_controller = PID(1, 0.1, 0.05, 0)
         self.right_rear_pid_controller = PID(1, 0.1, 0.05, 0)
         self.left_rear_pid_controller = PID(1, 0.1, 0.05, 0)
+
+        # Gaussian kernel
+        self.basis = self.gkern(kernlen=50)
```
Comment (acoulombe) [inline drive_control/src/drive_control_node.py:33]: Have 50 as a variable everywhere

File: drive_control/src/steering.py [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
     def steering_control(self, vR, wR): # R = rover
-        temp=wR*self.L/2
-        wLeft=(vR+temp)/self.rWheel * 1000
-        wRight=(vR-temp)/self.rWheel * 1000
-        val = [wLeft, wRight]                       
-        return val
-
+        
+        # Manually caps the linear and angular velocities.
+        if wR > 3.0:
```
Comment (acoulombe) [inline drive_control/src/steering.py:10]: Have min/max as variable that is changeable with input or helper functions

**Review by acoulombe — CHANGES_REQUESTED** [review]
Few things for Adam to fix

**Review by acoulombe — APPROVED** [review]

**Review by Shaswata79 — APPROVED** [review]

**Pull request merged**